skrub/_dataframe/_common.py

```diff
@@ -314,12 +315,22 @@ def _concat_horizontal_polars(*dataframes):
     return pl.concat(dataframes, how="horizontal")
 
 
+def is_column_list(obj):
```

Comment:
This will also accept `tuple` but I guess that we can silently support it.

Reply:
yes any sequence will do but `is_column_list` is a bit simpler name than `is_column_sequence` and resembles `to_column_list`